Sources/DVR/SessionDataTask.swift

```diff
@@ -45,6 +45,10 @@ final class SessionDataTask: URLSessionDataTask {
 
     override func resume() {
         
+        guard session != nil else {
+            fatalError("This shouldn't happen")
+        }
+        
```

Comment:
This isn't necessary, since if you dereference `session` and it's nil it will crash with a fatalError anyhow, but give a better error message.

Reply:
Arguably they shouldn't be using `!` anywhere, but this guard only makes sense if Session is optional

Reply:
This was necessary when testing for a weird error I encounter but it isn't anymore. I'll remove it